rdmo/management/assets/js/api/MultiSiteApi.js

```diff
@@ -0,0 +1,12 @@
+
+import BaseApi from 'rdmo/core/assets/js/api/BaseApi'
+
+class MultiSiteApi extends BaseApi {
+
+  static UpdateElementAction(element, action) {
+      const model_api_path = element.model.replace('.', '/').concat('s')
```

Comment:
This should be done using the objects in `constants/elements.js`.

Reply:
MultiSiteApi is removed altogether